src/Microsoft.Azure.WebJobs.Host/Config/DirectInvokeString.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Config
+{
+    // A short string serialization used to rehydrate the trigger. Sometimes this is the contents, and sometimes it's a moniker.  For example:
+    // For Blob, the DirectInvokeString is the blob path; whereas the String is the blob contents. (because blobs can be MB large) 
+    // For queue, both the DirectInvokeString and String are the queue contents. (because queue messages are generally small)
+    class DirectInvokeString
+    {
+        public string Value { get; set; }
+
+        public static DirectInvokeString New(string value)
```

Comment:
What is the benefit of this static New function, over a ctor? E.g. `new DirectInvokeString(...)` is more common/natural than `DirectInvokeString.New(...)` right?

Reply:
Also, this is just a wrapper around a string with no added behavior - why is this needed?